src/module/system/check/dialog.ts

```diff
@@ -29,14 +29,18 @@ export class CheckModifiersDialog extends Application {
     ) {
         // The title often has HTML in it: get the base text
         const title = ((): string => {
+            // add actor name to title to differentiate rolls
+            const actorName = context.actor?.name.trim() ?? "";
+
             const maybeWithHTML = context.title?.trim() || check.slug;
-            if (!maybeWithHTML.includes("<")) return maybeWithHTML.trim();
+            if (!maybeWithHTML.includes("<"))
```

Comment:
Might be better if these used localizable format strings rather than a concat.

Reply:
Done!